tools/myexample2/macros.xml

```diff
@@ -0,0 +1,12 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <yield/>
```

Comment:
there should be

`<requirement type="package" version="1.2">seqtk</requirement>` in here